docs/application/dotnet/samples/tizen-samples.md

```diff
@@ -31,6 +31,18 @@
 			<p>SecureRepository is simple application that allows you to securely store data, certificates and cryptographic keys using C# Tizen.Security.SecureRepository API.</p>
 			</td>
 		</tr>
+		<tr>
+			<td>
+			<p><img alt="" height="360" src="media/w4voicememo.png" width="360" /></p>
+			</td>
+			<td>
+			<p><strong>VoiceMemo</strong></p>
+			<p>The Voice Memo sample application demonstrates how you can create an application using Xamarin.Forms API and Tizen FX APIs.</p>
+			<p><a href="https://developer.tizen.org/ko/development/sample/native/UI/%28Circle%29_Voice_Memo" target="_blank">Native version</a></p>
+			<p><a href="https://developer.tizen.org/ko/development/sample/web/UI/Voice_Memo_UI" target="_blank">Web version</a></p>
+			<p>Comming soon</p>
```

Comment:
Question : It looks like we already have links to sample for native and web. So, for what is this "Comming Soon"?

Reply:
The links are for Native, Web application sample, C# is not yet ready.

Reply:
I see. Thanks for the description. I missed that point.